jdaviz/configs/specviz/helper.py

```diff
@@ -90,7 +90,8 @@ def get_spectral_regions(self):
 
         for name, reg in regions.items():
             unit = reg.meta.get("spectral_axis_unit", u.Unit("Angstrom"))
-            spec_reg = SpectralRegion.from_center(reg.center.x * unit, reg.width * unit)
+            spec_reg = SpectralRegion.from_center(reg.center.x * unit,
+                                                  (reg.width / 2) * unit)
```

Comment:
This should be taken care of in the `SpectralRegion.from_center` method (whose doc string is clearly inconsistent with its behavior), which is supposed to take a center and width value (and should do the halving there).

We can _probably_ let this slide for now, but we really ought to open an issue on the specutils repo to fix it at the root, and perhaps a comment here stating when that issue is solved to come fix this.

Reply:
That's a quick fix on the specutils side, I think - I can go ahead and open a PR there this morning and ditch the factor of two here (and in my moment maps plugin as well).

Reply:
Removed the factor of two here, specutils fix in https://github.com/astropy/specutils/pull/710